src/components/Modal/Wrapper.vue

```diff
@@ -48,11 +48,20 @@
           />
           {{ symbols.tokenOut }}
         </UiButton>
-        <div
-          v-text="$t('ethBuffer')"
-          class="text-yellow text-center mt-3"
-          v-if="!etherLeft"
-        />
+        <div v-if="balance.address !== 'xdai'">
+          <div
+            v-text="$t('xdaiBuffer')"
+            class="text-yellow text-center mt-3"
+            v-if="!etherLeft"
+          />
+          <div v-if="balance.address === 'xdai'">
+            <div
+              v-text="$t('xdaiBuffer')"
```

Comment:
Isn't this doing the same thing in the above condition?

Reply:
Yes, you're right. I'll check in a change to address this.